### ISSUE TITLE
Regression: Playstation Home 1.86 dev build hangs at loading screen (#8835)

### ISSUE THREAD

**Issue by ghost** [RPCS3/rpcs3#9092]
## Quick summary
Hanging at loading screen.

## Details

At boot, the Playstation Home 1.86 offline build hangs when going offline. It was working in the version: v0.0.12-10858-8c8048f0_win64

#### 1. RPCS3's log.

[PS Home log.zip](https://github.com/RPCS3/rpcs3/files/5397712/PS.Home.log.zip)

#### 2. Screenshots of the problem.

![Capture](https://user-images.githubusercontent.com/26351275/96372453-1bc2ff80-1167-11eb-8f49-1628970467b7.PNG)

#### 3. Please provide your system configuration:
OS : Windows 10 64bits
CPU: I9 9900K
GPU: Intel UHD 630
Driver version : 27.20.100.8783

#### 4. Important note:

Sorry KD-11 I didn't see your previous message but yes, unfortunately a lot of graphics issues remain, it would be awsome to look at those after the regression is fixed.

Linked issue: https://github.com/RPCS3/rpcs3/issues/8161

**Comment by AniLeo** [comment]
Missing what version regressed it

**Comment by atomic83GitHub** [comment]
It have regressed after the version 0.0.11-10669-f8d2d8ca

**Comment by AniLeo** [comment]
The next version is 0.0.11-10670 which literally just changes a log message, not possible that one caused the regression

**Comment by atomic83GitHub** [comment]
Arg, I will have to retest then. Do you know which one after those introduced a major changes in the core or rsx?

**Comment by AniLeo** [comment]
Many versions in between - and you can't know if it was a major change that broke it
Best to just use the normal binary searching method

**Comment by atomic83GitHub** [comment]
Where can I download the versions in between?

**Comment by AniLeo** [comment]
https://rpcs3.net/builds

**Comment by atomic83GitHub** [comment]
Beeing a bug squad in pcsx2 I have the habit of doing that :) , I might find it rather quickly.

**Comment by atomic83GitHub** [comment]
This is the real build that introduced the issue: rpcs3-v0.0.12-10858-8c8048f0_win64

**Comment by bevanweiss** [comment]
@atomic83GitHub best situation is if you can post a log from the immediately prior working build, showing it progressing past the point of the problem, and a similar log from the exact 'broken' build which shows the 'broken' problem.

Also, please don't use 'latest one' in reference to versions.  That information quickly becomes irrelevant. Write the actual version number.

In this particular situation, it would be worth you checking the validity of your files.
Immediately prior to the access violation is an ENOENT referring to a particular file.
The case of the path to this file (on your system) should match EXACTLY to what it is in the PS3 source.
You should also compare this particular reported error in the before and after logs, if it is in both, then it might just be a weirdness of the game.  If it is only present in the after, then that might be the cause of your problem.

**Comment by atomic83GitHub** [comment]
Here are the 2 logs from booth non working and working version:

https://github.com/RPCS3/rpcs3/files/5397712/PS.Home.log.zip

[PS Home working log.zip](https://github.com/RPCS3/rpcs3/files/5400568/PS.Home.working.log.zip)

And about the files, they does work fine and are fully accurate to what is on a real PS3.

**Comment by bevanweiss** [comment]
Can you attach a screenshot showing the file:
/dev_hdd0/game/NPIA00010/USRDIR/OFFLINECONTENT/OBJECTS/OBJECTCATALOGUE.XML

Because the working version opens this file, and the non-working version doesn't successfully open it, and then crashes when trying to memory map the file anyway... hence it's this file causing your crash.  And the major change around this is case-sensitive filesystem changes.

**Comment by atomic83GitHub** [comment]
There it is: 

[XML file.zip](https://github.com/RPCS3/rpcs3/files/5401790/XML.file.zip)

**Comment by bevanweiss** [comment]
Right... so the filename of that file you just sent appears to be "ObjectCatalogue.xml", and the filename that the PS3 is looking for is "OBJECTCATALOGUE.XML".
So when I asked you to check the case of the filenames... it seems that you did not do this very well.

I suspect that the rest of the path may have issues also (i.e. the OFFLINECONTENT folder, and the OBJECTS folder).

It's possible that the UDF case mapping isn't quite right in whatever you used to dump the disk.
What application did you use to do the dumping?

**Comment by atomic83GitHub** [comment]
I didn't do any dumping, this file is generated by installing the home dev pkg.

**Comment by bevanweiss** [comment]
Hmm... it might be a case for super @elad335
I think it's definitely a case sensitivity issue with that file, but if the .pkg was legitimate then I'd have expected it to extract files into the right locations/case context (unless there's a glitch with rpcs3 .pkg installation.. which I expect we'd have heard more about).

Have you tried loading this .pkg into a PS3?

**Comment by atomic83GitHub** [comment]
Yeah, loading this one to a real PS3 generate the same file.

**Comment by bevanweiss** [comment]
> Yeah, loading this one in a real PS3 generate the same file.

So it generates a file "OBJECTCATALOGUE.XML", with a capital O, capital B, capital E, capital J, capital C, capital T, capital C, capital A, capital T, capital A, capital L, capital O, capital G, capital U, capital E, decimal/dot, capital X, capital M, capital L?

**Comment by atomic83GitHub** [comment]
No, nothing is in capital.

**Comment by clienthax** [comment]
for reference https://github.com/RPCS3/rpcs3/commit/8c8048f0

**Comment by atomic83GitHub** [comment]
I am very sorry for this issue, it's not an emulator problem.

**Comment by I-Knight-I** [comment]
@AniLeo Apologies for the ping but I believe this issue needs to be re-opened. Would appreciate it if you could review my last comment and decide from there.

**Comment by digant73** [comment]
@I-Knight-I is that still an issue?
